### PR TITLE
Update french translation

### PR DESCRIPTION
**Pull request by dabou**
Add french translation in new text.

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -81,7 +81,7 @@ msgstr "Description"
 #: src/AppInfoView.vala:137
 #, fuzzy
 msgid "Show in Launcher"
-msgstr "Afficher dans le menu"
+msgstr "Afficher dans le Menu"
```
[the remark this answers is not in the document]
Comment (NathanBnm) [reply po/fr.po]: Yes, the first translations were good, not these ones.

File: po/fr.po [hunk 2]
```diff
@@ -29,11 +29,11 @@ msgstr "Aucune Application trouvée"
 #: data/com.github.donadigo.appeditor.appdata.xml.in:10
 msgid ""
 "Edit application entries shown in application menu and their properties."
-msgstr ""
+msgstr "Modifiez les raccourcis et leurs propriétés dans le menu des l'applications."
```
Comment (NathanBnm) [inline po/fr.po]: This should be "Modifiez les raccourcis et leurs propriétés dans le menu des applications."

File: po/fr.po [hunk 3]
```diff
@@ -29,11 +29,11 @@ msgstr "Aucune Application trouvée"
...
 
 #: data/com.github.donadigo.appeditor.appdata.xml.in:13
 msgid "Features include:"
-msgstr ""
+msgstr "Fonctionnalités incluent:"
```
Comment (NathanBnm) [inline po/fr.po]: This should be "Fonctionnalités incluses :"

File: po/fr.po [hunk 4]
```diff
@@ -47,7 +47,7 @@ msgstr "Nouvelle application"
 
 #: data/com.github.donadigo.appeditor.appdata.xml.in:17
 msgid "Change application's display name, icon and more"
-msgstr ""
+msgstr "Changer le nom d'une application, son icône, etc."
```
Comment (NathanBnm) [inline po/fr.po]: The final period shouldn't be there.

File: po/fr.po [hunk 5]
```diff
@@ -60,7 +60,7 @@ msgstr "Aucune Application trouvée"
 
 #: data/com.github.donadigo.appeditor.desktop.in:6
 msgid "Edit application entries"
-msgstr ""
+msgstr "Modifier les raccourcis "
```
Comment (NathanBnm) [inline po/fr.po]: This should be "Modifier les raccourcis d'application"

**Comment by donadigo** [comment]
Manually merged in https://github.com/donadigo/appeditor/commit/4553684303feae877234d32ed4cf3e039cb6417f. Thank you for your contribution!

**Pull request closed without merging**